SPARCore/GADArticle.m

```diff
@@ -0,0 +1,30 @@
+#import "GADArticle.h"
+
+@implementation GADArticle
+
++ (NSArray <GADArticle *> *) loadDummyArticles {
+
+    NSMutableArray <GADArticle *> *articleArray = [[NSMutableArray alloc] init];
+    NSArray <NSString *> *authorNames = @[@"Alex", @"Mitchell", @"Addi", @"Gould", @"Garrett", @"Wang", @"Alex2", @"French", @"Nathan", @"Gifford"];
+    
+    for (int i = 0; i < 10; i++) {
+        GADArticle *article = [[GADArticle alloc] init];
+        article.publicationId = @"8e031545-ba66-11e6-8193-a0999b05c023";
+        article.title = [NSString stringWithFormat:@"Testarticle %i", i];
+        article.authors = [NSArray arrayWithObjects:@{@"name": authorNames[i], @"email": @"addisemail.edu"}, nil];
+        article.series = @"ada28c7d-a49f-11e6-b9d3-a0999b05c023";
+        article.url = [NSURL URLWithString:@"http://www.thesandb.com/news/shacs-to-offer-funds-for-students-in-need.html"];
+        article.tags = [NSArray arrayWithObjects:@"amazing!", @"fantastic!", @"Computer science!", nil];
+        article.brief = [NSString stringWithFormat:@"This is very brief %i", i];
+        article.content = @"This is much less brief because it should be longer than the brief";
+        article.datePublished = [NSDate date];
+        article.issue = @"ae2f3b54-a49f-11e6-b1ab-a0999b05c023";
+        article.articleId = @"db7bc676-ba7d-11e6-8de8-a20f19d048b3";
+    
+        [articleArray addObject:article];
+    }
+    return articleArray;
+}
```

Comment:
Will prob look better if you add 1-2 spaces before }

Will leave you to decide